core/src/test/java/org/apache/spark/unsafe/map/AbstractBytesToBytesMapSuite.java

```diff
@@ -75,13 +75,6 @@
   @Mock(answer = RETURNS_SMART_NULLS) BlockManager blockManager;
   @Mock(answer = RETURNS_SMART_NULLS) DiskBlockManager diskBlockManager;
 
-  private static final class WrapStream extends AbstractFunction1<OutputStream, OutputStream> {
```

Comment:
you can eliminate the imports of `AbstractFunction1` and `OutputStream` after this